pkg/distributors/contracts/MerkleOrchard.sol

```diff
@@ -26,18 +26,19 @@ import "./interfaces/IDistributorCallback.sol";
 
 pragma solidity ^0.7.0;
 
-contract MerkleOrchard is IDistributor {
+contract MerkleOrchard {
     using SafeERC20 for IERC20;
 
     // Recorded distributions
-    // rewardToken > rewarder > distribution > root
+    // token > distributor > distribution > root
     mapping(IERC20 => mapping(address => mapping(uint256 => bytes32))) public trees;
-    // rewardToken > rewarder distribution > lp > root
+    // token > distributor distribution > lp > root
     mapping(IERC20 => mapping(address => mapping(uint256 => mapping(address => bool)))) public claimed;
-    // rewardToken > rewarder > balance
+    // token > distributor > balance
     mapping(IERC20 => mapping(address => uint256)) public suppliedBalance;
 
-    event RewardAdded(address indexed token, uint256 amount);
+    event DistributionAdded(address indexed token, uint256 amount);
+    event DistributionPaid(address indexed user, address indexed token, uint256 amount);
```

Comment:
Maybe 'sent' instead of 'paid'? Payment carries weird connotations I think. Like, you often pay for products or services. Paying a reward made sense, but I'd try to steer away from that here.

Reply:
will change this in a forthcoming PR